homework/index.js

```diff
@@ -30,17 +34,102 @@
     return elem;
   }
 
+  function renderError(error) {
+    const root = document.getElementById('root');
+    root.innerHTML = '';
+    createAndAppend('h1', root, { text: error.message, class: `alert-error` });
+  }
+
+  function addRow(tbody, label, value) {
+    const tr = createAndAppend('tr', tbody);
+    createAndAppend('td', tr, { text: `${label}:`, class: 'label' });
+    createAndAppend('td', tr, { text: value });
+    return tr;
+  }
+
+  function renderRepos(repo, div) {
+    fetchJSON(HYF_REPOS_URL, err => {
+      if (err) {
+        renderError(err);
+        return;
+      }
```

Comment:
What is the purpose of this `fetchJSON()` call? You already have the data you need in the `repo` argument. You can remove lines 51-55 and the corresponding closing brace and everything will still work.